Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/configuration/ExchangeManagementPortDescriptor.java

```diff
@@ -0,0 +1,25 @@
+package gov.hhs.fha.nhinc.configuration;
+
+import gov.hhs.fha.nhinc.exchangemanagement.EntityExchangeManagementPortType;
+import gov.hhs.fha.nhinc.messaging.service.port.SOAP12ServicePortDescriptor;
+
+/**
+ *
+ * @author ttang
+ */
+public class ExchangeManagementPortDescriptor extends SOAP12ServicePortDescriptor<EntityExchangeManagementPortType> {
+
+    private static final String WS_ADDRESSING_ACTION = "urn:gov:hhs:fha:nhinc:exchangemangement"
```

Comment:
this seems like this is a typo. urn:gov:hhs:fha:nhinc:exchangemangementEntityExchangeManagementRequest? I feel like there is a : missing in the middle of the namespace and the method.